test/apps/iodemo/io_demo.cc

```diff
@@ -548,6 +646,8 @@ int main(int argc, char **argv)
         return ret;
     }
 
+    std::srand(std::time(NULL));
```

Comment:
maybe add a seed to parameters and set like this as a default?

Reply:
done